contrib/backup/functions

```diff
@@ -47,16 +49,33 @@ function backup_files () {
   ln -sfn ${BACKUP_DIR}/${TIMESTAMP}_zammad_files.tar.gz ${BACKUP_DIR}/latest_zammad_files.tar.gz
 }
 
+function create_pgpassfile() {
+  export PGPASSFILE=$(mktemp)
+    chmod 600 "$PGPASSFILE"
+    cat <<EOT > "$PGPASSFILE"
+*:*:${DB_NAME}:${DB_USER}:${DB_PASS}
+EOT
+}
```

Comment:
This could probably use trap to do the clean-up automatically:

```suggestion
  trap "rm $PGPASSFILE" EXIT
}
```

Reply:
Excellent idea, didn't know this is possible. Added in 57be0b2.